src/hint-core.scss

```diff
@@ -13,7 +13,7 @@
 .hint, [data-hint] {
 	position: relative;
 	display: inline-block;
-
+	font-family: $fontFamily;
```

Comment:
`[data-hint]` doesn't need to have font family set. It needs to be on the `:after` element inside it.